packages/material-ui/src/FormLabel/FormLabel.js

```diff
@@ -69,6 +69,34 @@ const FormLabel = React.forwardRef(function FormLabel(props, ref) {
     states: ['color', 'required', 'focused', 'disabled', 'error', 'filled'],
   });
 
+  if (!ref) {
+    ref = React.useRef();
+  }
```

Comment:
https://app.circleci.com/pipelines/github/mui-org/material-ui/13321/workflows/92f688be-6317-4e3b-9d27-caadb64a34c8/jobs/152563/parallel-runs/0/steps/0-111

Not sure how to handle this actually, any hints @oliviertassinari ?